lib/jetstream/pull_consumer.ex

```diff
@@ -247,47 +180,34 @@ defmodule Jetstream.PullConsumer do
     )
   end
 
-  defp create_stream(_conn, stream) when is_binary(stream), do: {:ok, stream}
-
-  defp create_stream(conn, stream) do
-    try do
-      struct!(Jetstream.API.Stream, stream)
-    rescue
-      _ -> {:error, :invalid_stream_settings}
-    else
-      stream -> Jetstream.API.Stream.create(conn, stream)
-    end
-  end
-
-  defp create_consumer(_conn, consumer) when is_binary(consumer), do: {:ok, consumer}
-
-  defp create_consumer(conn, consumer) do
-    try do
-      struct!(Jetstream.API.Consumer, consumer)
-    rescue
-      _ -> {:error, :invalid_consumer_settings}
-    else
-      consumer -> Jetstream.API.Consumer.create(conn, consumer)
-    end
-  end
-
   defp stream_name(stream) when is_binary(stream), do: stream
   defp stream_name(stream), do: stream.name
 
   defp consumer_name(consumer) when is_binary(consumer), do: consumer
   defp consumer_name(consumer), do: consumer.durable_name
```

Comment:
these functions are unnecessary now, given that stream & consumer are required to be binaries